test/run-pass/unsized-locals/unsized-index.rs

```diff
@@ -0,0 +1,21 @@
+// `std::ops::Index` has an `: ?Sized` bound on the `Idx` type param. This is
```

Comment:
Please move the test to `test/ui/...` and add `// compile-pass` to the top of the file;

Otherwise, r=me with green travis.

Reply:
done!

Reply:
The file ended up in `/test/ui/...` instead of `src/test/ui/...` :D Aim for https://github.com/rust-lang/rust/tree/master/src/test/ui/unsized-locals

Reply:
:man_facepalming:

Reply:
That explains why I had to use `-f` to add it. Should be better now

Reply:
Yeah; the rebase went wrong tho :P